### PR TITLE
Only start mongodb for ms-output; fix logic of config files

### PR DESCRIPTION
**Pull request by amaltaro**
This PR fixes a few commits recently pushed in, such as:
https://github.com/dmwm/CMSKubernetes/commit/0251c3a8cef30e0226c8818c51406e990a14873f
https://github.com/dmwm/CMSKubernetes/commit/5f195f62de26e5979401ac06dc63feba226f151c
https://github.com/dmwm/CMSKubernetes/commit/86f83f3fbca3a4571e54bec2d8f19dd3a18d232a

In short, it fixes:
* only start mongodb service when reqmgr2ms is running the `ms-output` component
* fix logic of deployment/services_config configuration files for reqmgr2ms (and dbsmigration), we were deleting all the config files every time it evaluated to true

@muhammadimranfarooqi @vkuznet please review. I will not manage to test these changes today/tomorrow.

### CODE REVIEW

File: docker/dbsmigration/run.sh [hunk 1]
```diff
@@ -42,19 +42,18 @@ fi
 
 # use service configuration files from /etc/secrets if they are present
 cdir=/data/srv/current/config/$srv
-# check if /etc/secrets area contain at least one dbs configuration files
 cfiles="DBSDef.py DBSMigrate.py DBSGlobalReader.py DBSGlobalWriter.py DBSPhys03Reader.py DBSPhys03Writer.py Migration.py"
+pushd $cdir
```
Comment (vkuznet) [inline docker/dbsmigration/run.sh:46]: Alan, this PR should not change logic of another deployment, and the changes your provide are not correct. They will disallow the use case when we do not have these files in config area, i.e. when we want to run service with default (image provided) config files. Please revert the changes

File: docker/reqmgr2ms/run.sh [hunk 2]
```diff
@@ -40,21 +40,20 @@ if [ -f /etc/secrets/hmac ]; then
...
 fi
 
-# use service configuration files from /etc/secrets if they are present
+### For this specific reqmgr2ms service, we need to replace the default
+# configuration files, by those provided under the proper services_config dir
 cdir=/data/srv/current/config/$srv
-# check if /etc/secrets area contain at least one dbs configuration files
 cfiles="config-monitor.py config-output.py config-transferor.py"
+pushd $cdir
+rm $cfiles
```
Comment (vkuznet) [inline docker/reqmgr2ms/run.sh:48]: The logic is incorrect here. If you rm all files it means that your POD will not start the service if you k8s do not provide config files. You only need to remove them ONLY if at least one file is present in the k8s configuration area, and leave all files to allow default behavior.

File: docker/dbsmigration/run.sh (hunk 1 above)
Comment (amaltaro) [reply docker/dbsmigration/run.sh:46]: That's a good point. I didn't consider the use case where we want to run the service with the default image configuration. 
I can revert these changes, but note that your implementation will only work for a *single* configuration file provided in services_config. If someone wants to provide 2 or more files, and remove whatever comes in the image, it will not work.

File: docker/reqmgr2ms/run.sh (hunk 2 above)
Comment (amaltaro) [reply docker/reqmgr2ms/run.sh:48]: That's a valid use case, let me update it.

**Comment by amaltaro** [comment]
Valentin, can you please have another look?
The DBS changes were removed (even though I pointed a flaw in the original code) and I changed reqmgr2ms to keep original configs in case nothing gets provided under /etc/secrets area
